pkg/receive/handler.go

```diff
@@ -682,6 +692,22 @@ func (h *Handler) RemoteWrite(ctx context.Context, r *storepb.WriteRequest) (*st
 	}
 }
 
+// relabel relabels the time series labels in the remote write request.
+func (h *Handler) relabel(wreq *prompb.WriteRequest) {
+	if len(h.options.RelabelConfigs) > 0 {
```

Comment:
Let's invert this check to make the code go from top to bottom instead of left to right:
```go
if len(h.options.RelabelConfigs) == 0 {
  return
}
...
```